src/cashlib/libsecp256k1-config.h

```diff
@@ -0,0 +1,163 @@
+/* src/libsecp256k1-config.h.  Generated from libsecp256k1-config.h.in by configure.  */
+/* src/libsecp256k1-config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
why do we need to have this file if it is generated by the `configure` process?

Reply:
I'll remove that comment.  The android config.h is not generated by configure.  I don't run configure at all since it seems to be incompatible (or at least a huge waste of time making it so) with the gradle/cmake android build process.